### PR TITLE
Automatic deletion of jobs older than a given time

### PR DESCRIPTION
**Pull request by Chris7**
This will delete jobs older than a given age

### CODE REVIEW

**Comment by Chris7** [comment]
@mfitzp  Didn't you create something like this before? I can't seem to find it...

File: wooey/tasks.py [hunk 1]
```diff
@@ -183,3 +184,26 @@ def get_valid_file(cwd, name, ext):
...
     return (stdout, stderr)
+
+@celery_app.task(base=WooeyTask)
+def cleanup_wooey_jobs(**kwargs):
+    from datetime import timedelta, datetime
+    from .models import WooeyJob
+    # get the list of jobs that need deleting
+    cleanup_settings = wooey_settings.WOOEY_JOB_EXPIRATION
+    anon_settings = cleanup_settings.get('anonymous')
+    now = datetime.now()
```
Comment (mfitzp) [inline wooey/tasks.py]: Should this use [timezone aware datetime objects](https://docs.djangoproject.com/en/1.9/topics/i18n/timezones/#naive-and-aware-datetime-objects)? e.g.

    from django.utils import timezone
    now = timezone.now()

**Comment by mfitzp** [comment]
@Chris7 I though I had too yes but no sign of it in my local repo (maybe this was pre-merge?) 

Looks fine except for the comment on `from django.utils import timezone; timezone.now()` instead of `datetime.now()` for timezone awareness.

File: wooey/tasks.py (hunk 1 above)
Comment (Chris7) [inline wooey/tasks.py]: Good call. For some reason I thought Django was depracating their timezone in utils. Guess not!

**Pull request merged**